decidim-meetings/app/views/decidim/meetings/admin/meetings/index.html.erb

```diff
@@ -18,7 +18,7 @@
             <th><%= t("models.meeting.fields.start_time", scope: "decidim.meetings") %></th>
             <th><%= t("models.meeting.fields.end_time", scope: "decidim.meetings") %></th>
             <th><%= t("models.meeting.fields.closed", scope: "decidim.meetings") %></th>
-            <% if Decidim.geocoder.present? %>
+            <% if Decidim::Map.available?(:static, :geocoding) %>
```

Comment:
The star in front of the `*categories` parameter means that it takes all passed arguments as an array. The signature allows an unlimited amount of parameters to be passed to the method.

This double check is not 100% necessary but I thought it would make sense here because:
- The geocoding information is not available unless the `:geocoding` map utility is available
- The static map could not be displayed unless the `:static` map utility is available

Therefore, if only the `:static` map is available, it does not make sense to display the static map if the geocoding information is not available.

On the other hand, if only the `:geocoding` utility is available, it does not make sense to display the static map if the service provider does not offer such static map service.